### PR TITLE
Docs part 2.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: docs/en/getting-started.md [hunk 1]
```diff
@@ -79,3 +79,31 @@ Once you have a database connection instance, you can execute a **SQL** query by
...
 2. [Bind parameters](/docs/en/queries/bind-parameters.md)
 3. [Call one of the SQL execute method to execute the command](/docs/en/queries/execute-command.md)
+
+
+## Quoting Table and Column Names
+
+When writing database-agnostic code, properly quoting table and column names is often a headache because different databases have different name quoting rules. To overcome this problem, you may use the following quoting syntax introduced by [Yii DB](https://github.com/yiisoft/db):
+
+- `[[column name]]`: enclose a column name to be quoted in double square brackets.
+- `{{table name}}`: enclose a table name to be quoted in double curly brackets.
```
Comment (darkdef) [inline docs/en/getting-started.md:89]: {{%table_name}}  - need recommended add percent before name for use prefix of tables

File: docs/en/query-builder.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Db\Query\Query;
+
+/** @var ConnectionInterface $db */
+
+$rows = (new Query($db))
+    ->select(['id', 'email'])
+    ->from('user')
```
Comment (darkdef) [inline docs/en/query-builder.md:24]: from('{{%user}}')

and need add example with correct work with alias
['alias' => '{{%table_name}}']

File: docs/en/query/from.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// SELECT * FROM `user`
+$query->from('user');
+```
+
+You can specify the **table(s)** being selected from in either a string or an array. The table names may contain **schema prefixes and/or table aliases**, like you do when writing **raw SQL statements**.
+
+For example.
+
+```php
+$query->from(['public.user u', 'public.post p']);
```
Comment (darkdef) [inline docs/en/query/from.md:17]: need add "we not recommed this method - use array format" -

File: docs/en/getting-started.md (hunk 1 above)
Comment (darkdef) [reply docs/en/getting-started.md:89]: and need use {{%table_name}} in all documentations. It's recommended example of usage

**Pull request merged**